dandiapi/zarr/tests/utils.py

```diff
@@ -0,0 +1,25 @@
+import hashlib
+import os
+from pathlib import Path
+
+import faker
+from zarr_checksum.generators import ZarrArchiveFile
```

Comment:
This is sort of a criticism of `zarr-checksum`, but having `ZarrArchive` and `ZarrArchiveFile` interleaved in code is confusing. I'm bad at names, but `ZarrFileDescriptor` or something might be better.